packages/utils/src/__tests__/validation/validatorFunctions.test.js

```diff
@@ -5,20 +5,24 @@
 
 import { allValuesAreNumbers } from '../../validation';
 
-describe('validationFunctions', () => {
+describe('validatorFunctions', () => {
   describe('allValuesAreNumbers', () => {
-    it.each([
-      ['fails if a not given an object', null, ''],
-      ['fails if a value is not a number', { hi: 'hello' }, "Value 'hi' is not a number: 'hello'"],
-    ])('%s', (_, object, expectedError) => {
-      expect(() => allValuesAreNumbers(object)).toThrowError(expectedError);
+    it('fails if a not given an object', () => {
```

Comment:
Reverted to non-parameterized tests in this commit - seems more readable to me in this case